packages/mux-player/package.json

```diff
@@ -96,24 +96,25 @@
   "dependencies": {
     "@mux/mux-video": "0.20.2",
     "@mux/playback-core": "0.25.2",
-    "media-chrome": "~3.2.5"
+    "media-chrome": "~4.1.0",
+    "player.style": "^0.0.7"
```

Comment:
try upgrading to 0.0.8 for fixing the dup node_modules issue

Reply:
Done! see commit below for affect